core/AssetManager/UIAssetFetcher/JScriptUIAssetFetcher.php

```diff
@@ -102,11 +102,15 @@ private function addUmdFilesIfDetected($plugins)
         foreach ($plugins as $plugin) {
             $devUmd = "plugins/$plugin/vue/dist/$plugin.development.umd.js";
             $minifiedUmd = "plugins/$plugin/vue/dist/$plugin.umd.min.js";
-
-            if (Development::isEnabled() && is_file(PIWIK_INCLUDE_PATH . '/' . $devUmd)) {
-                $this->fileLocations[$plugin] = $devUmd;
-            } else if (is_file(PIWIK_INCLUDE_PATH . '/' . $minifiedUmd)) {
-                $this->fileLocations[$plugin] = $minifiedUmd;
+            $umdSrcFolder = "plugins/$plugin/vue/src";
+
+            // in case there are dist files but no src files, which can happen during development
+            if (is_dir($umdSrcFolder)) {
```

Comment:
During development it can be possible for there to be a dist/ folder but no src/ folder (happens when checking out different branches where one branch doesn't have a vue folder I believe).